server/src/main/java/com/objectcomputing/checkins/services/teammembers/TeamMember.java

```diff
@@ -0,0 +1,79 @@
+package com.objectcomputing.checkins.services.teammembers;
+
+import java.sql.Date;
```

Comment:
It looks like this import is unused. Can we remove this?

Reply:
Done